R/derive_advs_params.R

```diff
@@ -255,6 +255,15 @@ compute_map <- function(diabp, sysbp, hr = NULL) {
 #'
 #'   Permitted Values: character value
 #'
+#' @param constant_height HEIGHT parameter is constant
+#'
+#'   If the HEIGHT value is considered constant (e.g. measured once at screening),
+#'   the values of BSA will be calculated for each WEIGHT value.
```

Comment:
I would rephrase to: "To calculate BSA at every visit at which WEIGHT is collected using the patient's constant HEIGHT, set to TRUE. Otherwise, to calculate BSA only for the first instance in which HEIGHT and WEIGHT are collected, set to FALSE.